adhoc/workload.yaml

```diff
@@ -42,6 +42,7 @@ services:
         --key-file /shared_data/keys/workload.priv \
         --rate ${RATE:-1} \
         --urls $$TARGETS \
+        --batch-size ${BATCH:-1} \
```

Comment:
This should be in a separate commit, with a justification for the change in the commit message.

Reply:
Ok

Reply:
I see this change is in a separate commit: _Allow limiting number of transactions in a batch for workload_